Cargo.toml

```diff
@@ -83,13 +87,17 @@ version = "0.11.5"
 optional = true
 
 [dependencies.tokio]
-version = "~0.2.18"
+version = "1.0.1"
 features = ["io-util", "sync", "macros"]
 
 [dependencies.tokio-rustls]
-version = "0.13.0"
+version = "0.22.0"
 features = ["dangerous_configuration"]
 
+[dependencies.tokio-util]
```

Comment:
This provides compatibility between futures versions of AsyncRead/AsyncWrite and the new tokio versions, which we need for our stream implementation